Nodejs/Product/Nodejs/Project/ImportWizard/ImportSettings.cs

```diff
@@ -36,16 +38,18 @@ internal class ImportSettings : DependencyObject
                 ".ejs",
                 ".styl",
                 ".xml",
-                ".ts",
                 Jade.JadeContentTypeDefinition.JadeFileExtension,
                 Jade.JadeContentTypeDefinition.PugFileExtension
             }.Select(x => "*" + x));
 
-        private bool _isAutoGeneratedProjectPath;
+        private const string TypeScriptExtensions = ";*.ts;*.tsx";
```

Comment:
>TypeScriptExtensions [](start = 29, length = 20)

These seem more like "filters" than "extensions".  It might also be helpful to add a comment about the leading semicolon.

Reply:
sure